lifelines/tests/utils/test_utils.py

```diff
@@ -218,7 +219,7 @@ def test_datetimes_to_durations_will_handle_dates_above_multi_fill_date():
 
 def test_datetimes_to_durations_will_handle_dates_above_multi_fill_date():
     start_date = ["2013-10-08", "2013-10-09", "2013-10-10"]
-    end_date = ["2013-10-10", None, None]
+    end_date = ["2013-10-10", None, "2013-10-20"]
```

Comment:
To be clear, if you reverted your change in `datetimes_to_durations`, this test would fail, right?

Reply:
Yes. This fails without the change.